vc-verifier/kotlin/vcverifier/src/main/java/io/mosip/vercred/vcverifier/utils/Util.kt

```
@@ -148,6 +149,38 @@ object Util {
+
+        require(value.type == CBORType.Integer) {
+            "$name must be a CBOR integer (NumericDate)"
+        }
+
+        return value.AsInt64()
+    }
+
+    fun httpGet(url: String): String? {
+        return try {
+            restTemplate.getForObject(url, String::class.java)
+        } catch (e: Exception) {
```

detekt (1.23.8)

[warning] The caught exception is swallowed. The original exception could be lost.

(detekt.exceptions.SwallowedException)